temporal-test-server/src/main/java/io/temporal/internal/testservice/StateMachines.java

```diff
@@ -846,7 +872,10 @@ private static RetryState attemptNexusOperationRetry(
       ctx.onCommit(
           (historySize) -> {
             data.retryState = nextAttempt;
-            data.nextAttemptScheduleTime = ctx.currentTime().getSeconds();
+            data.isBackingOff = true;
```

Comment:
You should reset this to false when the backoff timer fires.

Reply:
I added it to where the timer actually fires here:
https://github.com/temporalio/sdk-java/pull/2258/files#diff-83c1231c017389012549bc7181908362062be65ffabc65ae405e7ad3cdedaa31R2284

Reply:
I just saw it now. Thanks.